internal/flink/command_artifact.go

```diff
@@ -0,0 +1,65 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type flinkArtifactSerializedOutOut struct {
+	Name           string `serialized:"name"`
+	Id             string `serialized:"id"`
+	ConnectorClass string `serialized:"connector_class"`
+	ContentFormat  string `serialized:"content_format"`
+}
+
+// TODO: https://confluentinc.atlassian.net/browse/FRT-334
+// This is reuse existing custom connector plugin api for flink udf management for EA customer only
+//
+//	aka, `ConnectorClass` will return version ID for EA
+//	For flink GA, flink team will have public API to do so
+type flinkArtifactHumanOut struct {
+	Name           string `human:"Name"`
+	Id             string `human:"Plugin ID"`
+	ConnectorClass string `human:"Version ID"`
+	ContentFormat  string `human:"Content Format"`
+}
```

Comment:
These structs can be combined into one since all types are strings

Reply:
i will keep them as they are as it looks more readable. wdyt?

Reply:
Let's combine them to make it easier to update them in the future and more closely align with our other commands

Reply:
updated